modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -1652,6 +1652,8 @@ FeatureFloodCount::FeatureData::merge(FeatureData && rhs)
   // Logical AND here to combine flags (INACTIVE & INACTIVE == INACTIVE, all other combos are CLEAR)
   _status &= rhs._status;
 
+  _intersects_boundary |= _intersects_boundary;
```

Comment:
Wat?

Reply:
This is in the merge method of two partial features. If any part of the feature intersects the boundary, then the merged feature intersects the boundary. I suppose I can add a comment 😄

Reply:
... are you gas lighting me now? It is the same variable on both sides of the `|=` operator.

```
a = a | a
```
will never change `a`'s value!

Reply:
Did you mean 

```
_intersects_boundary |= rhs._intersects_boundary 
```

maybe?

Reply:
Yes, that's the right one... Oops... I gave this fix to Fande, then mis-typed it here. Perhaps I should work up a test that will fail on 2 procs (current test takes several procs before it fails without this line)